src/jit/compiler.h

```diff
@@ -4223,6 +4227,8 @@ class Compiler
     {
         SPK_Unknown,       // Invalid value, never returned
         SPK_PrimitiveType, // The struct is passed/returned using a primitive type.
+        SPK_EnclosingType, // Like SPK_Primitive type, but used for return types that
+                           //  are not a power of two byte size.
```

Comment:
Comment should be more like:
```
// Like SPK_Primitive type,  but used for return types that 
// require a primative type temp that is larger that the struct size.
// Currently used for structs of sizes: 3,5,6 or 7 bytes
```

Reply:
I am still not sure if I'm keeping this -- let me work through arm32 / arm64 and see how I feel once that's fixed.